styles.css

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
This file may not be showing because:
under mood-selector you have a border but not whether is solid (not sure if this is a big deal)
They a divs so they should naturally show in a column since they are block elements.
Under .Happy, you put a border colour but no border